tutorials/basics_tutorial.ipynb

```diff
@@ -32,13 +32,14 @@
    "source": [
```

Comment:
<p>It looks like the cell outputs from here onward are no longer displayed in ReviewNB. I could imagine the outputs would be useful for someone scanning the document and not wanting to run it themselves. Maybe run all the cells and recommit? Hopefully that will fix the issue.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/solvation-analysis/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Done